packages/components/src/DataList/__tests__/DataList.selectable.test.tsx

```diff
@@ -0,0 +1,91 @@
+import { screen, waitFor } from "@testing-library/react";
+import * as pom from "../__POM__/DataList.selectable.pom";
```

Comment:
I've never seen this, but it's really interesting. Is it something that we should consider using more often in our tests?